app/views/layouts/_footer.html.erb

```diff
@@ -33,54 +33,7 @@
 
     <div class="small-12 medium-4 column social">
       <div class="text-right">
-        <ul>
```

Comment:
I think the changes on this file are not needed since we have a custom view for the footer at (`app/views/layouts/_footer.html.erb`), so I guess this file won't be ever rendered. Correct me if I am wrong. 🙏🏼

Reply:
You're right :smile:. Fixed!